app/src/main/java/com/cupcake/chickenmasala/data/dataSource/DataSourceImpl.kt

```diff
@@ -79,21 +77,22 @@ class DataSourceImpl(private val context: Application) : DataSource {
         }
     }
 
-    private fun toFormattedTime(time: Int): String {
```

Comment:
when use to prefix name function i expected use function as mapper but u use function as normal function, so remove to from name and keep it formatTime